src/openloco/widget.cpp

```diff
@@ -51,6 +51,15 @@ namespace openloco::ui::widget
         call(0x004CF3EB, regs);
     }
 
+    void sub_4CF487(gfx::drawpixelinfo_t* dpi, const window* window, const widget_index index)
```

Comment:
This draws the viewport center button

Reply:
Ah, I thought I'd already renamed this one. Indeed.